substrate-node/pallets/pallet-runtime-upgrade/src/lib.rs

```diff
@@ -22,7 +22,7 @@ pub mod pallet {
     impl<T: Config> Pallet<T> {
         #[pallet::call_index(0)]
         // Give same weight as set_code() wrapped extrinsic from frame_system
-        #[pallet::weight((T::BlockWeights::get().base_block, DispatchClass::Operational))]
+        #[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
```

Comment:
Can you explain this change, what it does?

Reply:
see definitions `base_block` and `max_block` definitions here
https://github.com/paritytech/polkadot-sdk/blob/fee810a5ea177f95ddf13ea51270b84f1e117bae/substrate/frame/system/src/limits.rs#L199

But it changed after polkador upgrade so I preferred to update the weight to `<T as Config>::SystemWeightInfo::set_code()` same weight as called function in `frame_system` pallet here https://github.com/paritytech/polkadot-sdk/blob/fee810a5ea177f95ddf13ea51270b84f1e117bae/substrate/frame/system/src/lib.rs#L610